source/CMakeLists.txt

```diff
@@ -188,89 +188,61 @@ add_custom_target(spirv-tools-build-version
    DEPENDS ${SPIRV_TOOLS_BUILD_VERSION_INC})
 set_property(TARGET spirv-tools-build-version PROPERTY FOLDER "SPIRV-Tools build")
 
-add_subdirectory(comp)
-add_subdirectory(opt)
-add_subdirectory(link)
+add_library(${SPIRV_TOOLS} "")
 
-set(SPIRV_SOURCES
-  ${spirv-tools_SOURCE_DIR}/include/spirv-tools/libspirv.h
+include(GenerateExportHeader)
+generate_export_header(${SPIRV_TOOLS}
+  BASE_NAME spirv-tools)
 
-  ${CMAKE_CURRENT_SOURCE_DIR}/util/bitutils.h
```

Comment:
.h files are important to keep in the source list so that they are shown in IDEs like visual studio.